### PR TITLE
Notebooks update [skip ci]

### PR DESCRIPTION
**Pull request by chesterxgchen**
Fixes # .

### Description
Based on QA team's suggestion, 
1. we are repeating the POC and NVFLARE setup at beginning of the few notebooks 
2. Previously the POC setup has to be down as extra setup as terminal. Now, we use background bash and is able to overcome this. So end-to-end POC setup can be done from notebook. 
3. Fed Stats Notebooks  (two), Hello-world and POC setups are updated

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/advanced/federated-statistics/df_stats.ipynb [hunk 1]
```diff
@@ -1,115 +1,49 @@
...
-   "metadata": {
-    "tags": []
-   },
-   "source": [
-    "## Install NVFLARE (from Pypi) "
+    "You can also follow this [notebook](../../../nvflare_setup.ipynb) Notebook to setup.\n",
+    "\n",
+    "Just to quick recap the nvflare install and POC installation steps in the followings\n",
+    "\n",
+    "**NVLARE Installation**"
```
Comment (nvkevlu) [inline examples/advanced/federated-statistics/df_stats.ipynb:22]: NVLARE -> NVFLARE

File: examples/advanced/federated-statistics/df_stats.ipynb [hunk 2]
```diff
@@ -300,29 +239,62 @@
...
+     "source_hidden": true
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%bash --bg  \n",
+    "\n",
+    "nvflare poc --start -ex admin\n",
+    "echo \"steep for few seconds for system to start\"\n",
```
Comment (nvkevlu) [inline examples/advanced/federated-statistics/df_stats.ipynb:284]: steep -> sleep

File: examples/advanced/federated-statistics/image_stats.ipynb [hunk 3]
```diff
@@ -250,24 +265,136 @@
...
+   "id": "19f98ff0-4ab1-4631-81a9-8e2ff2f0bcf8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%bash --bg  \n",
+    "\n",
+    "nvflare poc --start -ex admin\n",
+    "echo \"steep for few seconds for system to start\"\n",
```
Comment (nvkevlu) [inline examples/advanced/federated-statistics/image_stats.ipynb:313]: steep -> sleep

File: examples/tutorials/hello_world.ipynb [hunk 4]
```diff
@@ -15,12 +15,206 @@
     "\n",
     "Each example is self-contained. You can start from any example. But you must run through the 3 steps of each example in sequence.\n",
     "\n",
+    "## Prerequists\n",
```
Comment (nvkevlu) [inline examples/tutorials/hello_world.ipynb:18]: Prerequisites

**Review by nvkevlu — APPROVED** [review]
We can merge this, and I can polish the language and minor fixes in my PR

**Review by nvkevlu — APPROVED** [review]

File: examples/tutorials/hello_world.ipynb [hunk 5]
```diff
@@ -15,12 +15,206 @@
...
+   "id": "b627dddb-4924-41b1-a853-3ad3b74d2d2e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%bash --bg  \n",
+    "\n",
+    "nvflare poc --start -ex admin\n",
+    "echo \"steep for few seconds for system to start\"\n",
```
Comment (YuanTingHsieh) [inline examples/tutorials/hello_world.ipynb:127]: typo, steep should be sleep

**Pull request merged**